packages/vuetify/src/components/VOtpInput/VOtpInput.sass

```diff
@@ -0,0 +1,16 @@
+.v-otp-input-field
+  input
+    text-align: center
+  .v-input__control
+    .v-input__slot
+      background: #ffffff
+  input[type=number]::-webkit-outer-spin-button,
+  input[type=number]::-webkit-inner-spin-button
+    -webkit-appearance: none
+    margin: 0
+  input[type=number]
+    -moz-appearance: textfield
+.theme--dark.v-otp-input-field
```

Comment:
Use the theme mixin

Reply:
Resolved on https://github.com/vuetifyjs/vuetify/pull/12761/commits/a4efc9c65a2341c186aff274c235d8e5d85816cf#diff-8599e86758d0ad6e1a1bd6702ea3c2716d9adce26d048319608e5365b62b349f. 

Please confirm.

Reply:
See https://github.com/vuetifyjs/vuetify/blob/17bd263fed490177d634bc8421bdfa8dbf6dc1ef/packages/vuetify/src/components/VTextField/VTextField.sass#L4

Theme-related styles go in there and use colours from `$material`, which is in `_light.scss` and `_dark.scss`